src/bidiMapper/domains/script/realm.ts

```diff
@@ -110,13 +110,16 @@ export class Realm {
       }
     }
 
-    if (
-      cdpValue.result.type === 'object' &&
-      cdpValue.result.subtype === 'generator'
-    ) {
-      return {
-        type: 'generator',
-      };
+    if (cdpValue.result.type === 'object') {
```

Comment:
This check is  done only for the top-level objects, but not for the nested one. Traversing nested data is done in [`#deepSerializedToBiDi`](https://github.com/GoogleChromeLabs/chromium-bidi/pull/1042/files#diff-d7af9adf42c7a4f4040462b9ff23cae43df9de6a355ee3197df6d9462485c1c3R93), but there is no `type` and `subtype` data.